lib/guessinggame.rb

```diff
@@ -0,0 +1,27 @@
+num = Random.new.rand(1..100)
+puts 'Guess a number between 1 and 100'
+puts num
+
+solved = 'no'
+tries = 0
+until solved == 'yes' do
+  guess = gets.to_i
+
+      if guess<1 || guess>100||!guess.is_a?(Fixnum)
+        puts 'Guess A WHOLE number between 1 and 100.'
+      elsif num <guess
+
+        puts "The number is lower than #{guess}.  Guess again "
+      tries = tries + 1
+      elsif num > guess
+        puts "The number is higher than #{guess}.  Guess again "
+        tries = tries + 1
+      elsif num = guess
+        tries = tries + 1
```

Comment:
You can take out tries into the first `if` body so you increment the value once, instead of repeating it on each conditions